client/fe/config/config.js

```diff
@@ -75,22 +75,23 @@ class Config {
           /ol-.+/,
           /plyr-.+/,
           /uppy-.+/,
-          /color-.+/,
+          /^color-.+/,
```

Comment:
why that? Or other way round: shouldn't we add the `^` to (almost) all the entries?

Reply:
> shouldn't we add the ^ to (almost) all the entries?

This is probablyright but i did not want to risk breaking stuff, and i did not want to change it everywhere without cheching side effects... which would take some time for those big ones.